packages/CONTROLS_STATUS.md

```diff
@@ -0,0 +1,35 @@
+## Description
+This table shows the status of features in Fluent UI React Native.
+## Status of Features Table
+| Controls  | iOS      | Mac      | Android  | WinUI    |
+| --------- | -------- | -------- | -------- | -------- |
+| Avatar    | Prod     | Prod     | Preview  | Preview  |
+| Badge     | Backlog  | Backlog  | Backlog  | Backlog  |
+| Button    | Prod     | Prod     | Prod     | Preview  |
+| Callout   | Backlog  | Backlog  | Backlog  | Backlog  |
+| Checkbox  | Preview  | Preview  | Preview  | Preview  |
+| Dialog    | Backlog  | Backlog  | Backlog  | Backlog  |
+| Divider   | Prod     | Prod     | Prod     | Preview  |
+| Dropdown  | Backlog  | Backlog  | Backlog  | Backlog  |
+| FocusZone | Backlog  | -        | Backlog  | Backlog  |
+| Icon      | Prod     | Prod     | Prod     | Backlog  |
+| Image     | N\A      | N\A      | N\A      | N\A      |
+| Label     | Backlog  | Backlog  | Backlog  | Backlog  |
+| Link      | Preview  | Preview  | Prod     | Content  |
+| Menu      | Backlog  | -        | Backlog  | Backlog  |
+| RadioGroup| Backlog  | -        | Backlog  | Preview  |
+| Slider    | Backlog  | Backlog  | Backlog  | Backlog  |
+| Stack     | Backlog  | Backlog  | Backlog  | Backlog  |
+| Spinner   | Backlog  | Backlog  | Backlog  | Backlog  |
+| Text      | Prod     | Prod     | Prod     | Preview  |
+| TextField | Backlog  | Backlog  | Backlog  | Backlog  |
+| Toggle    | Backlog  | Backlog  | Backlog  | Backlog  |
+| Tooltip   | Backlog  | Backlog  | Backlog  | Backlog  |
+  
+### Statuses
+**Prod:** Ready for production.  
+**Preview:** Is partially developed.  
+**Backlog:** Development not yet started, but is an objective to develop.
+
+## Updating The Table
+If you've made a PR that changes the status of a feature, please update the status of that feature in the table.
```

Comment:
Re my comment earlier. A year ago we had an intern do basically this exact thing where every component's status was listed in it's test page on FluentTester. This is a redefinition that seems to be different, and not tied to code.

We should either remove the latter, or have this one tied to the info coming from there. Two sets of info feels more confusing imo.